eng/release/DiagnosticsReleaseTool/Common/BlobLayoutWorker.cs

```diff
@@ -0,0 +1,14 @@
+using System.IO;
+
+namespace ReleaseTool.Core
+{
+    public sealed class BlobLayoutWorker : PassThroughLayoutWorker
+    {
+        public BlobLayoutWorker(string stagingPath) : base(
+            shouldHandleFileFunc: static file => true,
```

Comment:
Does this mean that this always handles all files? I see that you relaxed the assumption that only one handler should handle things - instead you create cascading ones. I am OK with that, but is there a way to assert this is the last one?

Reply:
> Does this mean that this always handles all files?

Yes, because there isn't a good way to say what is a blob file and what isn't since it's just given a file path to figure everything out.

> I see that you relaxed the assumption that only one handler should handle things - instead you create cascading ones.

That assumption was not relaxed; it still remains. What I did change was allowing additional handlers to attempt to handle an already handled file, which was odd because then there was a validation check to make sure that a file wasn't handled more than once. So rather than allowing the bad behavior to occur and then logging an error when the bad behavior happens, I changed it so only one can handle it and the remaining handlers are ignored.

> I am OK with that, but is there a way to assert this is the last one?

Not sure, maybe remove it from the list and then explicitly add it as the fallback in the `Release` ctor.